examples/trusted_information_retrieval/module/rust/src/lib.rs

```diff
@@ -74,6 +75,54 @@ impl TrustedInformationRetrievalNode {
         // Compute distance.
         earth_radius * central_angle
     }
+
+    /// Create a gRPC client pseudo-Node.
+    fn get_grpc_client(&self) -> anyhow::Result<DatabaseClient> {
+        oak::grpc::client::Client::new_with_label(
+            &oak::node_config::grpc_client(&self.database_url),
+            &Label::public_untrusted(),
+        )
+        .map(DatabaseClient)
+        .ok_or(anyhow!("Couldn't create a gRPC client"))
+    }
+
+    /// Load a database subset defined by `offset` and the number of requested elements
+    /// (`page_size`).
+    fn list_database_entries(
+        &self,
+        offset: u32,
+        page_size: u32,
+    ) -> anyhow::Result<(Vec<PointOfInterest>, bool)> {
```

Comment:
Not sure the bool is relevant, just return things until empty or the number of entries is less than the page size.